api/v1alpha1/jwt_types.go

```diff
@@ -89,9 +89,35 @@ type ClaimToHeader struct {
 
 // JWTExtractor defines a custom JWT token extraction from HTTP request.
 type JWTExtractor struct {
+	// Headers represents a list of HTTP request headers to extract the JWT token from.
+	//
+	// +optional
+	Headers []JWTHeaderExtractor `json:"headers,omitempty"`
+
 	// Cookies represents a list of cookie names to extract the JWT token from.
 	// If specified, Envoy will extract the JWT token from the listed cookies and validate each of them.
 	// If any cookie is found to be an invalid JWT, a 401 error will be returned.
```

Comment:
should we move this invalid JWT string to top level `JWTExtractor`, since its valid for all fields now ?

Reply:
that's right, will move it to the top